lingetic-nextjs-frontend/app/components/PostHogProvider.tsx

```
@@ -0,0 +1,57 @@
+import { PostHogProvider as PHProvider, usePostHog } from "posthog-js/react";
+import { Suspense, useEffect } from "react";
+import { usePathname, useSearchParams } from "next/navigation";
+import assert from "@/utilities/assert";
+
+export function PostHogProvider({ children }: { children: React.ReactNode }) {
+  useEffect(() => {
+    assert(
+      process.env.NEXT_PUBLIC_POSTHOG_KEY != null,
+      "PostHog key is required"
+    );
+    posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)